strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/metadata/PypiArtifactMetadata.java

```diff
@@ -0,0 +1,244 @@
+package org.carlspring.strongbox.artifact.metadata;
+
+/**
+ * 
+ * @author ankit.tomar
+ *
+ */
+public class PypiArtifactMetadata
+{
+
+    private String comment = null;
```

Comment:
No need for all these explicit `null` declarations.

Reply:
Noted.